### ISSUE TITLE
suggestion: still allow players to see previous run after the winner has won the game

### ISSUE THREAD

**Issue by Alegria4486** [Lattyware/massivedecks#547]
nothing else to add, but your game is awesome and it works perfectly as intended!

**Comment by Lattyware** [comment]
Yeah, letting users see the game history at game end would make a lot of sense and not be hard to do. Sounds like a good idea.

**Comment by Alegria4486** [comment]
say, if it's alright for me to make another suggestion instead of making a new ticket... could it be possible to program the game to have blank white cards + the bot, and the bot just discards its blank cards as soon as it has them in its hand (then the Never Have I Ever option would be obligatory)? The people I play with keep having trouble choosing between the two features.

**Comment by Alegria4486** [comment]
oh, and a third suggestion. If I use my web browser's "screenshot the whole page" function, it does this
![screenshot](https://github.com/Lattyware/massivedecks/assets/32369634/f102a7ac-1b64-47d9-9cf4-a027833d05b4)
(the red rectangles are me censoring the names, it's not a glitch)
it doesn't happen when I screenshot what I can directly see without scrolling. Could the game have an option to save a picture of the whole game at the end?

**Comment by lhvy** [comment]
Probably make a different issue for each of these comments.

Regarding the bot + blank cards compatibility, I would be interested in fixing that once I have slightly more time next month.
